tests/tests/test-filter-api.ts

```diff
@@ -138,8 +133,25 @@ describeWithMoonbeam("Moonbeam RPC (EthFilterApi)", `simple-specs.json`, (contex
     await customRequest(context.web3, "eth_sendRawTransaction", [tx.rawTransaction]);
     return tx;
   }
+  let currentId;
```

Comment:
This seems to be shared across the tests. If that is the case, we should find a way to have the tests isolated

Reply:
I can red the currentId in each of the tests separately, but I guess it will have the same effect as reading it in a beforeEach statement right?

Reply:
I don't see where the currentId is set except on line 151 where it is set to 0.

Reply:
Thats right, it comes from a setup I had before (were I wasnt uninstalling the filters before each test). So we can always assume it will be 0 before the test starts